build.sbt

```diff
@@ -198,6 +198,7 @@ lazy val chisel = (project in file(".")).
   settings(
     mimaPreviousArtifacts := Set(),
     libraryDependencies += defaultVersions("treadle") % "test",
+    scalacOptions += "-P:chiselplugin:genBundleElements",
```

Comment:
I think we cannot enable this for regular Chisel (but can in the tests) for the same reasons we couldn't enable autoclonetype2, it'll cause issues for any Bundle extending a Bundle in `chisel3.util` (see https://github.com/chipsalliance/chisel3/releases/tag/v3.4.3)

Reply:
@jackkoenig can you point me to the place where this should go

Reply:
It just needs to be `Test / scalacOptions += "-P:chiselplugin:genBundleElements",`

Reply:
Will do